apps/api/src/app/workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase.ts

```diff
@@ -563,3 +611,19 @@ function mapAjvErrorToIssueType(error: ErrorObject): StepContentIssueEnum {
       return StepContentIssueEnum.MISSING_VALUE;
   }
 }
+
+function mapAjvErrorToMessage(error: ErrorObject<string, Record<string, unknown>, unknown>): string {
+  if (error.keyword === 'required') {
+    return `${_.capitalize(error.params.missingProperty)} is required`;
+  }
+  if (
+    error.keyword === 'pattern' &&
+    error.message?.includes('must match pattern') &&
+    error.message?.includes('mailto') &&
+    error.message?.includes('https')
+  ) {
+    return 'Invalid URL format. Must be a valid absolute URL, path, or valid variable';
```

Comment:
'Invalid URL format. Must be a valid absolute URL, path starting with /, or Novu {{variable}}';